src/main/secret/secret_manager.cpp

```diff
@@ -283,8 +282,8 @@ BoundStatement SecretManager::BindCreateSecret(CatalogTransaction transaction, C
 	}
 
 	BoundStatement result;
-	result.names = {"Success"};
-	result.types = {LogicalType::BOOLEAN};
+	result.names = {"Success", "Secret"};
```

Comment:
The `Success` field is mostly just a placeholder used for statements that don't have result sets - if we are going to return a result set from `CREATE SECRET` we might want to change this to something more descriptive (as I would expect !success to throw an error instead)